charts/opentelemetry-collector/README.md

```diff
@@ -56,11 +58,9 @@ By default collector has the following receivers enabled:
 - **traces**: OTLP, zipkin and jaeger (thrift and grpc).
 - **logs**: OTLP (to enable container logs, see [Configuration for Kubernetes container logs](#configuration-for-kubernetes-container-logs)).
 
-There are two ways to configure collector pipelines, which can be used together as well.
-
-### Basic top level configuration
+### Basic Top Level Configuration
 
-Default components can be removed with `null`.  When changing a pipeline, you must explicitly list all the components that are in the pipeline, including any default components.
+The Collector's configuration is set via the `config` section. Default components can be removed with `null`.  When changing a pipeline, you must explicitly list all the components that are in the pipeline, including any default components.
```

Comment:
This is applicable to any lists in the config, not pipelines only. I think it's important to call out

Reply:
Modified it to be more generic